tiledb/api/c_api/group/group_api.cc

```diff
@@ -693,6 +750,17 @@ capi_return_t tiledb_group_get_member_by_index(
       ctx, group, index, uri, type, name);
 }
 
+capi_return_t tiledb_group_get_member_by_index_v2(
+    tiledb_ctx_t* ctx,
+    tiledb_group_t* group,
+    uint64_t index,
+    tiledb_string_t** uri,
+    tiledb_object_t* type,
+    tiledb_string_t** name) TILEDB_NOEXCEPT {
+  return api_entry_context<tiledb::api::tiledb_group_get_member_by_index_v2>(
+      ctx, group, index, uri, type, name);
+}
+
```

Comment:
```suggestion
CAPI_INTERFACE(group_get_member_by_index_v2,
    tiledb_ctx_t* ctx,
    tiledb_group_t* group,
    uint64_t index,
    tiledb_string_t** uri,
    tiledb_object_t* type,
    tiledb_string_t** name) {
  return api_entry_context<tiledb::api::tiledb_group_get_member_by_index_v2>(
      ctx, group, index, uri, type, name);
}

```

It looks like this needs to be rebased and converted to the new CAPI_INTERFACE approach.